packages/mobile/src/screens/app-screen/AppScreen.tsx

```diff
@@ -25,6 +28,10 @@ export type AppScreenParamList = {
 const Tab = createBottomTabNavigator()
 
 export const AppScreen = () => {
+  const { updateRequired } = useUpdateRequired()
+
+  if (updateRequired) return <UpdateRequiredScreen />
```

Comment:
would it make sense to incorporate react-router here? and if so, maybe we move this to the root-screen where we check if logged-in? so we have three root branches?